src/Control/Email/Mailer.php

```diff
@@ -2,10 +2,10 @@
 
 namespace SilverStripe\Control\Email;
 
-use InvalidArgumentException;
-use SilverStripe\Control\HTTP;
-use SilverStripe\Core\Convert;
+use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Core\Object;
+use Swift_Mailer;
+use Swift_MailTransport;
```

Comment:
I second @kinglozzer's suggestion, however I don't think it's critical to fully-decouple from swiftmailer for the sake of the initial implementation. It's not a merge blocker. :)